src/Microsoft.AspNetCore/WebHost.cs

```diff
@@ -191,5 +191,40 @@ public static IWebHostBuilder CreateDefaultBuilder(string[] args)
 
             return builder;
         }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="WebHostBuilder"/> class with pre-configured defaults using typed Startup
+        /// </summary>
+        /// <typeparam name="T">Specify the startup type to be used by the web host.</typeparam>
```

Comment:
I would prefer using the same language as `UseStartup` https://github.com/aspnet/Hosting/blob/70a3017b12b4e9648f65acea2c9ad37e383763b0/src/Microsoft.AspNetCore.Hosting/WebHostBuilderExtensions.cs#L77